modules/ossm-member-roll-create.adoc

```diff
@@ -102,3 +101,41 @@ spec:
 ----
 +
 . Modify the default YAML to add your projects as `members`.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+[id="ossm-member-roll-create-member_{context}"]
+== Creating the {ProductName} members
+
+`ServiceMeshMember` resources are created by service mesh users who are not administrators. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they are not allowed to point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can modify the mesh users role binding in the control plane project to specify the users and groups that are granted access.
+
+The `ServiceMeshMember` adds the project to the `ServiceMeshMemberRoll` within the control plane project it references. 
+
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+spec:
+  controlPlaneRef:
+    namespace: control-plane-namespace
+    name: minimal-install
```

Comment:
In ossm-control-plane-deploy.adoc the name of the SMCP is `basic-install`, so the same name should be used here.